azerothcore/azerothcore-wotlk
# fix(Scripts/MoltenCore): Shazzrah should teleport to players

## Changes Proposed:
-  Shazzrah should teleport to players, not the other way around
-  

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/9121

## SOURCE:


## Tests Performed:

- Tested in-game.

https://user-images.githubusercontent.com/47818697/142033901-2cf47b64-f82f-49e6-ba1d-86ae3177a7fe.mp4

## How to Test the Changes:


1. go to shazzrah with two characters
2. notice the boss blinks into the character further away
3.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
